src/test/java/com/karankumar/bookproject/ui/book/BookFormTest.java

```diff
@@ -167,7 +183,7 @@ void formFieldsPopulated() {
     @EnumSource(EventType.class)
     void saveEventPopulated(EventType eventType) {
         // given
-        populateBookForm();
+        populateBookForm(READ, false, null);
 
```

Comment:
Is it worth having a separate method for populating form for an existing book? Passing null values may not be a good idea.

Reply:
Maybe like this:
```java
private void populateBookForm(ShelfName shelfName, boolean isInSeries) {
        populateBookFormWithExistingBook(shelfName, isInSeries, null);
    }

private void populateBookFormWithExistingBook(ShelfName shelfName, boolean isInSeries, Book maybePresentBook) {
    ...
    }
```
I also pass a null-Value here, but with two completely separate methods, the switch-part would be duplicated.